docs/api/expressions/top_level.md

```diff
@@ -2,29 +2,27 @@
 
 These methods and objects are available directly in the `ibis` module.
 
-## `NA`
-
-`NA` is the null scalar.
-
 ::: ibis.and_
 ::: ibis.array
 ::: ibis.asc
 ::: ibis.case
-::: ibis.coalesce
+::: ibis.expr.types.Value.coalesce
```

Comment:
Were these not showing up or otherwise misleading?

Reply:
they were rendering in a suboptimal way:

![2023-06-08_11-38](https://github.com/ibis-project/ibis/assets/3596999/d61ed093-f30f-4825-883c-42d9d2818983)